packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -294,7 +294,9 @@ private boolean isDeviceSupported() {
 
   private boolean canAuthenticateWithBiometrics() {
     if (biometricManager == null) return false;
-    return biometricManager.canAuthenticate() == BiometricManager.BIOMETRIC_SUCCESS;
+    int biometricStatus = biometricManager.canAuthenticate();
+    return biometricStatus == BiometricManager.BIOMETRIC_SUCCESS
+        || biometricStatus == BiometricManager.BIOMETRIC_STATUS_UNKNOWN;
```

Comment:
Can you explain why you are changing `canAuthenticateWithBiometrics`? If someone isn't enrolled, this method should return false, so this doesn't seem like a desirable change.

Reply:
if `biometricStatus == BiometricManager.BIOMETRIC_STATUS_UNKNOWN` we don't know someone is enrolled or not

Reply:
if `BIOMETRIC_STATUS_UNKNOWN` we can still authenticate with biometrics

Reply:
as we can see in [documentation](https://developer.android.com/reference/androidx/biometric/BiometricManager#BIOMETRIC_STATUS_UNKNOWN)

> ... Applications that wish to enable biometric authentication on affected devices may still call BiometricPrompt#authenticate() after receiving this status code...

Reply:
You've clipped out a critical part:
> Applications that wish to enable biometric authentication on affected devices may still call BiometricPrompt#authenticate() after receiving this status code **but should be prepared to handle possible errors.**

That means that "true" is no the correct return value here, because while it *might* be the case that biometric auth is available, it might also not. It's a different state. Reporting that would require API changes so that callers can decide whether to proceed or not.

Reply:
@stuartmorgan I want to fix [#96646](https://github.com/flutter/flutter/issues/96646). Do you have any idea?

Reply:
I don't understand the connection between this change and that issue. For instance, if your goal is to change handling of the "not enrolled case", why aren't you changing anything related to BIOMETRIC_ERROR_NONE_ENROLLED`?

You should clearly describe, either in the PR or in the issue, the specific flow whose behavior you are trying to change, how you determined that the change you are making is correct, and add thorough unit test coverage that shows (and asserts) the behaviors you are trying to create.